src/test/java/org/broadinstitute/consent/http/service/DataAccessRequestServiceTest.java

```diff
@@ -375,10 +368,52 @@ void validateProgressReport() {
     parentDar.setUserId(user.getUserId());
+    assertDoesNotThrow(() -> service.validateProgressReport(user, progressReport, parentDar));
+  }
+
+  @Test
+  void validateProgressReportWithCollaboratorsAndStaffInSameInstitution() {
+    User user = createUserWithPrerequisites();
+    DataAccessRequest progressReport = generateProgressReport();
+    progressReport.setDatasetIds(List.of(1, 2));
+    DataAccessRequestData progressReportData = progressReport.getData();
+    Collaborator collaborator1 = new Collaborator();
+    collaborator1.setEmail("1" + user.getEmail());
+    progressReportData.setInternalCollaborators(Collections.singletonList(collaborator1));
+    Collaborator collaborator2 = new Collaborator();
+    collaborator2.setEmail("2" + user.getEmail());
+    progressReportData.setLabCollaborators(Collections.singletonList(collaborator2));
+    DataAccessRequest parentDar = generateDataAccessRequest();
+    parentDar.setSubmissionDate(Timestamp.from(Instant.now()));
+    parentDar.setDatasetIds(List.of(1, 2, 3));
+    parentDar.setUserId(user.getUserId());
+    when(userDAO.findUserByEmail(collaborator1.getEmail())).thenReturn(user);
     when(institutionService.findInstitutionForEmail(any())).thenReturn(user.getInstitution());
     assertDoesNotThrow(() -> service.validateProgressReport(user, progressReport, parentDar));
   }
 
+  @Test
+  void validateProgressReportWithCollaboratorsAndStaffInInvalidInstitution() {
+    User user = createUserWithPrerequisites();
+    DataAccessRequest progressReport = generateProgressReport();
+    progressReport.setDatasetIds(List.of(1, 2));
+    DataAccessRequestData progressReportData = progressReport.getData();
+    Collaborator collaborator1 = new Collaborator();
+    collaborator1.setEmail("alice@1otherdomain.org");
+    progressReportData.setInternalCollaborators(Collections.singletonList(collaborator1));
+    Collaborator collaborator2 = new Collaborator();
+    collaborator2.setEmail("eve@yetanotherdomain.org");
+    progressReportData.setLabCollaborators(Collections.singletonList(collaborator2));
+    DataAccessRequest parentDar = generateDataAccessRequest();
+    parentDar.setSubmissionDate(Timestamp.from(Instant.now()));
+    parentDar.setDatasetIds(List.of(1, 2, 3));
+    parentDar.setUserId(user.getUserId());
+    when(userDAO.findUserByEmail(collaborator1.getEmail())).thenReturn(user);
+    when(institutionService.findInstitutionForEmail(any())).thenReturn(null);
+    BadRequestException badRequestException = assertThrows(BadRequestException.class, () -> service.validateProgressReport(user, progressReport, parentDar));
+    assertTrue(badRequestException.getMessage().contains("All listed personnel must share the same institutional affiliation.  The following list of roles and members must have email addresses associated with your institution: Internal Collaborator member: alice@1otherdomain.org, Lab staff member: eve@yetanotherdomain.org"));
```

Comment:
This is a great case for using `assertThat()` over `assertTrue()`. If you use `assertThat()`, it will print out the both strings so you can see why the `contains()` is failing.
```suggestion
    assertThat(badRequestException.getMessage(), containsString("All listed personnel must share the same institutional affiliation.  The following list of roles and members must have email addresses associated with your institution: Internal Collaborator member: alice@1otherdomain.org, Lab staff member: eve@yetanotherdomain.org"));
```